operations/mimir/ruler-query-path.libsonnet

```diff
@@ -0,0 +1,80 @@
+// Deployment of an isolated, dedicated query path for ruler remote evaluation.
+{
+  _config+:: {
+    ruler_query_path_enabled: false,
+    ruler_remote_evaluation_enabled: false,
+
+    // Note: There is no option to disable query-scheduler for the ruler path.
```

Comment:
Then please add a check to ensure query-scheduler is enabled when remote rule evaluation is used. An example of check is:

```
check_querier_autoscaling: if !$._config.simple_scalable_deployment_enabled || !$._config.autoscaling_querier_enabled then null else
    error 'please set autoscaling_querier_enabled to false when using Mimir simple scalable deployment',
```

Reply:
It doesn't matter if the `query-scheduler` is disabled. I can re-word the comment:

```
     // Note: There is no option to disable ruler-query-scheduler.
```